### PR TITLE
Show all logging output on GUI

### LINKED ISSUE #173: Log data shown in GUI is different to what is in train-log files.
EG, GUI triphone stage shows:
```
tree-info exp/tri1/tree
tree-info exp/tri1/tree
fstcomposecontext --context-size=3 --central-position=1 --read-disambig-syms=data/lang/phones/disambig.int --write-disambig-syms=data/lang/tmp/disambig_ilabels_3_1.int data/lang/tmp/ilabels_3_1.4464
fstisstochastic data/lang/tmp/CLG_3_1.fst
make-h-transducer --disambig-syms-out=exp/tri1/graph/disambig_tid.int --transition-scale=1.0 data/lang/tmp/ilabels_3_1 exp/tri1/tree exp/tri1/final.mdl
fstrmepslocal
fstrmsymbols exp/tri1/graph/disambig_tid.int
fstdeterminizestar --use-log=true
fsttablecompose exp/tri1/graph/Ha.fst data/lang/tmp/CLG_3_1.fst
fstminimizeencoded
fstisstochastic exp/tri1/graph/HCLGa.fst
add-self-loops --self-loop-scale=0.1 --reorder=true exp/tri1/final.mdl
```

However the train log is like:
```
===== MONO TRAINING =====
steps/train_mono.sh --nj 1 --cmd run.pl data/train data/lang exp/mono
steps/train_mono.sh: Initializing monophone system.
steps/train_mono.sh: Compiling training graphs
steps/train_mono.sh: Aligning data equally (pass 0)
steps/train_mono.sh: Pass 1
etc
```

Maybe good if the GUI shows both?

### PR DESCRIPTION
**Pull request by mattchrlw**
Closes #173. Before it was only showing select info, now it shows the entire output of `train.log`.

image

### CODE REVIEW

**Review by nicklambourne — APPROVED** [review]
LGTM

**Review by nicklambourne — APPROVED** [review]

**Review by benfoley — CHANGES_REQUESTED** [review]
Please check that both engines have these fixes for consistency.

**Comment by mattchrlw** [comment]
I've modified the logic a little bit, please have a look now

File: elpis/engines/kaldi/objects/model.py [hunk 1]
```diff
@@ -236,13 +236,11 @@ def train():
 
                 # Run the command, log output. Also redirect Kaldi sterr output to log. These are often not errors :-(
                 try:
-                    stage_process = run(f"cd {local_kaldi_path}; stages/{stage} >> {stage_log_path}")
-                    with open(stage_log_path, 'a+') as file:
-                        print('stdout', stage_process.stdout, file=file)
-                        print('stderr', stage_process.stderr, file=file)
-                        print('done', file=file)
+                    stage_process = run(f"cd {local_kaldi_path}; stages/{stage} &> {stage_log_path}")
```
Comment (mattchrlw) [inline elpis/engines/kaldi/objects/model.py:239]: `&>` ensures both stderr and stdout are logged as they come. This removes the need to print stdout and stderr to file and then print it out.

**Review by nicklambourne — APPROVED** [review]
LGTM if you can please unstage the yarn changes

**Review by benfoley — APPROVED** [review]

**Pull request merged**